avm/res/service-fabric/cluster/main.bicep

```diff
@@ -304,9 +303,9 @@ resource serviceFabricCluster 'Microsoft.ServiceFabric/clusters@2021-06-01' = {
       : null
     certificate: !empty(certificate)
       ? {
-          thumbprint: contains(certificate, 'thumbprint') ? certificate.thumbprint : null
-          thumbprintSecondary: contains(certificate, 'thumbprintSecondary') ? certificate.thumbprintSecondary : null
-          x509StoreName: contains(certificate, 'x509StoreName') ? certificate.x509StoreName : null
+          thumbprint: certificate.?thumbprint ?? ''
```

Comment:
This is required so I guess the empty value is not an option. We can probably go for (to be tested)
```suggestion
          thumbprint: certificate.thumbprint
```

Reply:
When certificate is conditional, this results in an error message 
![image](https://github.com/Azure/bicep-registry-modules/assets/37806411/57f39657-bab9-4fcb-9391-7cf1dfabd461)

Reply:
I see, still the value is required and we are here accepting that to be an empty string. I won't block the PR on it but ask to keep that in mind to be investigated in a next version. You can resolve this comment.